guide/additional-info/changes-in-v13.md

```diff
@@ -0,0 +1,769 @@
+
+### MessageManager
+
+Methods were added to `MessageManager` to provide API support for uncached messages.
+
+#### MessageManager#crosspost
+
+`channel.messages.crosspost('765432109876543210')`
+
+Equivalent to `message.crosspost()`.
+
+#### MessageManager#edit
+
+`channel.messages.edit('765432109876543210', content, options)`
+
+Equivalent to `message.edit(content, options)`.
+
+#### MessageManager#pin
+
+`channel.messages.pin('765432109876543210', options)`
+
+Approximately equivalent to `message.pin(options)` but does not resolve to a Message.
+
+#### MessageManager#react
+
+`channel.messages.react('765432109876543210', emoji)`
+
+Approximately equivalent to `message.react(emoji)` but does not resolve to a MessageReaction.
+
+#### MessageManager#unpin
+
+`channel.messages.unpin('765432109876543210', options)`
+
+Approximately equivalent to `message.unpin(options)` but does not resolve to a Message.
+
+### NewsChannel
+
+#### NewsChannel#addFollower
+
+`channel.addFollower(channel)`
+
+Provides API support for bots to follow announcements in other channels.
+
+#### NewsChannel#setType
+
+`channel.setType('text')`
+
+Allows conversion between NewsChannel and TextChannel.
+
+### Permissions
+
+#### Permissions#STAGE_MODERATOR
+
+Static bitfield representing the permissions required to moderate a Stage Channel.
+
+### Role
+
+#### Role#tags
+
+Tags for roles belonging to bots, integrations, or premium subscribers.
+
+### RoleManager
+
+#### RoleManager#botRoleFor
+
+`guild.roles.botRoleFor(user)`
+
+Gets the managed role a bot created when joining the guild, if any.
+
+#### RoleManager#premiumSubscriberRole
+
+`guild.roles.premiumSubscriberRole`
+
+Gets the premium subscriber (booster) role for the Guild, if any.
+
+### StageChannel
+
+Stage Channels are now supported.
+
+### Sticker
+
+Stickers are now supported.
+
+### TextChannel
+
+#### TextChannel#awaitMessageComponentInteraction
+
+Promisified collector to wait for and collect a single component interaction in a TextChannel
+
+#### TextChannel#createMessageComponentInteractionCollector
+
+New Collector class for collecting component interactions in a TextChannel
+
+#### TextChannel#setType
+
+`channel.setType('news')`
+
+Allows conversion between TextChannel and NewsChannel.
+
+### Webhook
+
+#### Webhook#deleteMessage
+
+Webhooks can now delete messages that were sent by the Webhook.
+
+#### Webhook#editMessage
+
+Webhooks can now edit messages that were sent by the Webhook.
+
+#### Webhook#fetchMessage
+
+Webhooks can now fetch messages that were sent by the Webhook.
+
+#### Webhook#sourceChannel
+
+#### Webhook#sourceGuild
+
+Webhooks can now have a `sourceGuild` and `sourceChannel` if the message is being crossposted.
+
+### Util
+
+#### Util#resolvePartialEmoji
+
+The new `Util#resolvePartialEmoji` method attempts to resolve properties for a raw emoji object from input data, without the use of the Discord.js Client class or it's EmojiManager.
```

Comment:
```suggestion
The new `Util#resolvePartialEmoji` method attempts to resolve properties for a raw emoji object from input data, without the use of the Discord.js Client class or its EmojiManager.
```
Small typo. `it's` is a contraction for `it is`, not the possessive belonging to `it` (`its`) which is what this sentence seeks to utilise.